match/lib/match/importer.rb

```diff
@@ -31,7 +31,9 @@ def import_cert(params, cert_path: nil, p12_path: nil, profile_path: nil)
         readonly: params[:readonly],
         username: params[:username],
         team_id: params[:team_id],
-        team_name: params[:team_name]
+        team_name: params[:team_name],
+        api_key_path: params[:api_key_path],
+        api_key: params[:api_key]
```

Comment:
@joshdholtz just following up from #17344. I was able to fix the issue by adding the following here.
```
...
        team_name: params[:team_name],
        s3_bucket: params[:s3_bucket]
      })
```

If I understand this correctly these params will also need to be added. 

```
          s3_region: params[:s3_region],
          s3_access_key: params[:s3_access_key],
          s3_secret_access_key: params[:s3_secret_access_key],
          s3_object_prefix: [:s3_object_prefix],
```

Reply:
Ah cool! I will add this into this PR and get this out on Thursday-ish 😊 Thanks for finding the fix ❤️